src/client/clientmap.cpp

```diff
@@ -100,6 +100,28 @@ ClientMap::ClientMap(
 
 }
 
+void ClientMap::updateCamera(const v3f &pos, const v3f &dir, f32 fov, const v3s16 &offset)
```

Comment:
general advice: vectors are only a couple bytes so using a constref for them doesn't make much sense

Reply:
I was hoping that it would allow passing arguments in registers, rather than pushing on stack.

Reply:
While a pointer definitely fits in a register I think having to dereference it is still worse than reading everything close together from the stack.
Anyway, there are still a few more `const v3s16 &` and `const v3f &`

Reply:
Done